networking/private-networking.html.md

```diff
@@ -298,3 +298,61 @@ To list all the tunnels set up for an organization, run `fly wireguard list`. Yo
 #### Remove a tunnel
 
 To remove a tunnel, run `fly wireguard remove`. You can specify the organization and tunnel name on the command line or be prompted for both.
+
+### Troubleshooting
+
+Having trouble connecting to a Fly.io hosted resource? The Private Network VPN feature makes it easy for a local machine to establish a fast, reliable connection to a Fly.io organization. However, when you can't connect to something, it's helpful to establish a baseline of what is, or what is not, working.
```

Comment:
Suggest re-wording this slightly. Maybe just remove the second sentence and the "however" from the third sentence? We don't need to sell the feature here and we'll usually try to avoid saying something is "easy" (at least in straight-up tech docs and especially in troubleshooting docs).